cvmfs/cvmfs.cc

```diff
@@ -1331,6 +1358,11 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     attribute_value = string(VERSION) + "." + string(CVMFS_PATCH_LEVEL);
   } else if (attr == "user.pubkeys") {
     attribute_value = mount_point_->signature_mgr()->GetActivePubkeys();
+  } else if (attr == "user.repo_counters") {
+    attribute_value = mount_point_->catalog_mgr()->GetRootCatalog()->
```

Comment:
The CSV maps needs to be extracted before `fuse_remounter_->fence()->Leave()`, otherwise there is a race.

Reply:
Actually, there is not a critical race.